subsys/lorawan_lbm/Kconfig

```
@@ -0,0 +1,341 @@
+	default n
+
+config LORA_BASICS_MODEM_FUOTA_MAX_SIZE_OF_FRAGMENTS
+	int "The maximum fragments size for LBM FUOTA"
+	depends on LORA_BASICS_MODEM_FUOTA_MAX_SIZE_OF_FRAGMENTS_CONFIG
+
+config LORA_BASICS_MODEM_FUOTA_MAX_FRAGMENTS_REDUNDANCY_CONFIG
+	bool "Allow configuration of the maximum fragments redundancy for LBM FUOTA"
+	default n
+
+config LORA_BASICS_MODEM_FUOTA_MAX_FRAGMENTS_REDUNDANCY
+	int "The maximum fragments redundancy for LBM FUOTA"
+	depends on LORA_BASICS_MODEM_FUOTA_MAX_FRAGMENTS_REDUNDANCY_CONFIG
+
+
+choice
+	prompt "LoRaWan services for Firmware Upgrade Over The Air (FUOTA) version to use"
+	depends on LORA_BASICS_MODEM_FUOTA
+	config LORA_BASICS_MODEM_FUOTA_V1
+		bool "1"
+	config LORA_BASICS_MODEM_FUOTA_V2
+		bool "2"
+endchoice
+
+config LORA_BASICS_MODEM_FUOTA_VERSION
+	int
+	default 1 if LORA_BASICS_MODEM_FUOTA_V1
+	default 2 if LORA_BASICS_MODEM_FUOTA_V2
+
+
+config LORA_BASICS_MODEM_FUOTA_FMP
+	bool "LoRaWAN FUOTA Firmware Management Package support"
+	depends on LORA_BASICS_MODEM_FUOTA
+	default y
+
+config LORA_BASICS_MODEM_FUOTA_MPA
+	bool "LoRaWAN FUOTA Multi-Package Access support"
+	depends on LORA_BASICS_MODEM_FUOTA
+	default n
+
+#-----------------------------------------------------------------------------
+# LoRaCloud related options
+#-----------------------------------------------------------------------------
+
+config LORA_BASICS_MODEM_ALMANAC
+	bool "Cloud Almanac Update service (only applicable for lr1110 and lr1120 targets)"
+	depends on DT_HAS_SEMTECH_LR1110_ENABLED || DT_HAS_SEMTECH_LR1120_ENABLED
+	default n
+
+config LORA_BASICS_MODEM_STREAM
+	bool "Stream service"
+	default n
+
+config LORA_BASICS_MODEM_LFU
+	bool "Large File Upload service"
+	default n
+
+config LORA_BASICS_MODEM_DEVICE_MANAGEMENT
+	bool "Device Management service"
+	default n
+
+config LORA_BASICS_MODEM_GEOLOCATION
+	bool "Geolocation service"
+	depends on DT_HAS_SEMTECH_LR1110_ENABLED || DT_HAS_SEMTECH_LR1120_ENABLED
+	default n
+
+config LORA_BASICS_MODEM_STORE_AND_FORWARD
+	bool "Store And Forward service"
+	default n
+
+#-----------------------------------------------------------------------------
+# Relay and beacon options
+#-----------------------------------------------------------------------------
+
+# TODO:
+config LORA_BASICS_MODEM_BEACON_TX
+	bool "Beacon transmission support"
+	default n
+
+# TODO:
+config LORA_BASICS_MODEM_RELAY_TX
+	bool "Relay TX feature"
+	default n
+
+# TODO: not released yet
+# config LORA_BASICS_MODEM_RELAY_RX
+# 	bool "Relay RX support"
+# 	default n
+
+#-----------------------------------------------------------------------------
+# Miscellaneous options
+#-----------------------------------------------------------------------------
+
+config LORA_BASICS_MODEM_NUMBER_OF_STACKS
+	int "Number of LoRa Basics Modem stacks (only 1 is supported for now)"
+	default 1
+	help
+	 Number of stacks to be used by the modem library.
```

Since 1 is the only valid value, this should not be user-assignable:

```suggestion
config LORA_BASICS_MODEM_NUMBER_OF_STACKS
	int
	default 1  # only 1 is supported for now
	help
	 Number of stacks to be used by the modem library.
```